stories/Button/Button.jsx

```diff
@@ -1,153 +1,215 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-
-import theme from './theme';
-import skins from './skins';
-
-const fontSize = ({ size }) => {
-  const sizes = {
-    xsmall: '12px',
-    small: '12px',
-    medium: '16px',
-    large: '20px',
-    xlarge: '24px',
-  };
-  return `font-size: ${sizes[size] || sizes.medium};`;
-};
-
-const padding = ({ size }) => {
-  const paddings = {
-    xsmall: '0 12px',
-    small: '0 12px',
-    medium: '0 16px',
-    large: '0 16px',
-    xlarge: '0 16px',
-  };
-
-  return `padding: ${paddings[size] || paddings.medium};`;
-};
-
-const height = ({ size }) => {
-  const heights = {
-    xsmall: '24px',
-    small: '32px',
-    medium: '40px',
-    large: '48px',
-    xlarge: '56px',
-  };
-
-  return `height: ${heights[size] || heights.medium};`;
-};
+import { shadow, hexToRgba } from '../shared';
```

Comment:
`hexToRgba` is not being used in this file.

Reply:
Done